router-tests/testenv/testenv.go

```diff
@@ -717,7 +711,7 @@ func createTestEnv(t testing.TB, cfg *Config) (*Environment, error) {
 		}
 	}
 
-	waitErr := e.WaitForServer(ctx, e.RouterURL+"/health/live", 100, 10)
+	waitErr := e.WaitForServer(ctx, e.RouterURL+"/health/ready", 100, 10)
```

Comment:
Can the route be configured? If yes, should we read the config or hardcode this here?

Reply:
We can configure it, the router just has a fallback to a default value and I suppose that's why we never configured it in the testenv

```go
	// Default values for health check paths

	if r.healthCheckPath == "" {
		r.healthCheckPath = "/health"
	}
	if r.readinessCheckPath == "" {
		r.readinessCheckPath = "/health/ready"
	}
	if r.livenessCheckPath == "" {
		r.livenessCheckPath = "/health/live"
	}

```

Reply:
@jensneuse did you mix it up? This is testenv.

Reply:
My point was, if we ever write a test to validate the path of ready, live and health can be changed, this WaitFor will fail.